features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -81,9 +81,9 @@
 #define DEFAULT_ACCEPTMBOX_SIZE     8
 
 #ifdef LWIP_DEBUG
-#define TCPIP_THREAD_STACKSIZE      1024*2
+#define TCPIP_THREAD_STACKSIZE      1200*2
```

Comment:
Even more into the black magic area then - providing a uniform EMAC interface is one thing, but how on earth can anyone know the total combined stack requirements of stack + driver? Would need to either get quite deep there, or just set "hopefully big enough" and pray.